docs/nidigital/class.rst

```diff
@@ -1483,10 +1483,12 @@ get_site_pass_fail
 
             :type site_list: str
 
-            :rtype: list of bool
+            :rtype: { int: bool, int: bool, ... }
             :return:
 
 
+                    Dictionary where each key is the site number and the value is the pass or fail result
```

Comment:
It may be obvious but it wouldn't hurt to say that True is for pass and False is for fail. Or simply use Pass/Fail which I think makes it clear that Pass is True.